docs/charter.rst

```diff
@@ -0,0 +1,90 @@
+
+The Committee will:
+
+- Advise NF staff infrastructure team on how to administer account credentials and access
+- Make ongoing recommendations to improve and enhance available infrastructure
+- Act as a liaison between sponsored projects and the NF Infrastructure Staff, (who works with vendors)
+- Have a representative at monthly project meetings
+- Triage new request, e.g. is a request a common need or one off
+- Present best practices from various projects
+- Review its charter annually and recommend changes to the board for review
+
+Duration/Type of Committee
+==========================
+
+The committee will be a standing committee.
+
+Committee Composition
+---------------------
+
+Membership
+^^^^^^^^^^
+
+Membership will reflect the NumFOCUS Diversity Statement by welcoming and encouraging participation by people of all backgrounds and identities and promoting and sustaining a culture that values mutual respect, acceptance, and learning.
+
+The committee will consist of at least 5 members from NumFOCUS at-large community, with the following constraints:
+
+- 1 member is a project contributor to a NumFOCUS sponsored project
+- No 2 members are related in employment or NumFOCUS project leadership committee
+- 1 member is a NumFOCUS Staff member
+
+Officers
+^^^^^^^^
+
+The Vice President will call for scheduled meetings, determine the agenda for meetings, and chair the meetings.
+The Secretary will record all meeting minutes and submit quarterly reports to the board of directors.
+
+Selection
+^^^^^^^^^
+
+Members will be selected by the committee's officers or current members using a rating system.
+The Vice President may be selected by the board of directors or elected annually by the members of the committee.
+The secretary will be elected by the committee's members.
+
+Terms
+^^^^^
+
+Committee members will serve a one-year term, at which time they will have the option for unlimited renewals.
+The Vice President will serve a one-year term with the option to renew for one additional term.
+The Secretary will serve a one-year term with the option to renew for one additional term.
+
+Meetings
+========
+
+The committee will meet at least once every two months.
+A majority of the committee members shall constitute a quorum.
+
+Record Keeping and Reporting
+============================
+
+All committee records will be kept in https://github.com/numfocus/infrastructure
+
+A report on the state of project digital infrastructure including all committee actions will be submitted to the board of directors quarterly on or before the 15th of March, June, September, and December.
+
+Officers and Members
+====================
+
+2023-2024 Term
+---------------
+
+* Vice-President: Andy R. Terrel
+* Secretary: Nabil Freij
+* Members: Katrina Riehl, Irfan Alibay, Mridul Seth,
```

Comment:
```suggestion
* Members: Katrina Riehl, Irfan Alibay, Mridul Seth,
* Observing Members: Nicole Foster, Matt Moore, Arliss Collins
```
@aterrel would this be correct or should I merge them into one members list?

Reply:
Yes looks correct